app/src/main/java/com/infomaniak/swisstransfer/ui/images/icons/Bin.kt

```diff
@@ -42,88 +42,82 @@ val AppIcons.Bin: ImageVector
 
         _bin = Builder(
             name = "Bin",
-            defaultWidth = 16.0.dp,
-            defaultHeight = 16.0.dp,
-            viewportWidth = 16.0f,
-            viewportHeight = 16.0f,
+            defaultWidth = 24.0.dp,
```

Comment:
<img width="1062" alt="Capture d’écran 2024-11-29 à 16 42 14" src="https://github.com/user-attachments/assets/249a0acc-b082-49a6-9d2f-94150a110fa7">

Is it me or the bin is blurry when zoomed in ? I might need glasses.

Reply:
It seems that it depends of the device.
I have the same feeling as you on a Freeform.

But on Pixel 7, it's perfectly smooth.